chalicelib/checks/audit_checks.py

```diff
@@ -468,6 +468,61 @@ def check_status_mismatch(connection, **kwargs):
     return check
 
 
+@check_function(id_list=None)
+def check_opf_status_mismatch(connection, **kwargs):
+    check = init_check_res(connection, 'check_opf_status_mismatch')
+
+    opf_set = ('search/?type=ExperimentSet&other_processed_files.title%21=No+value&field=status'
+               '&field=other_processed_files&field=experiments_in_set.other_processed_files')
+    opf_exp = ('search/?type=ExperimentSet&other_processed_files.title=No+value'
+               '&experiments_in_set.other_processed_files.title%21=No+value'
+               '&field=experiments_in_set.other_processed_files&field=status')
+    opf_set_results = ff_utils.search_metadata(opf_set, ff_env=connection.ff_env)
+    opf_exp_results = ff_utils.search_metadata(opf_exp, ff_env=connection.ff_env)
+    results = opf_set_results + opf_exp_results
+    files = 'search/?type=FileProcessed&field=status'
+    file_results = ff_utils.search_metadata(files + '&experiment_sets.uuid!=No+value', ff_env=connection.ff_env)
+    file_results += ff_utils.search_metadata(files + '&experiments.uuid!=No+value', ff_env=connection.ff_env)
+    file_results += ff_utils.search_metadata(files + '&status=deleted&status=replaced&experiment_sets.uuid!=No+value',
+                                             ff_env=connection.ff_env)
+    file_results += ff_utils.search_metadata(files + '&status=deleted&status=replaced&experiments.uuid!=No+value',
+                                             ff_env=connection.ff_env)
+    status_dict = {f['@id']: f['status'] for f in file_results}
+    check.full_output = {}
+    for result in results:
+        titles = [item['title'] for item in result.get('other_processed_files', [])]
+        titles.extend([item['title'] for exp in result.get('experiments_in_set', [])
+                       for item in exp.get('other_processed_files', [])])
+        titles = list(set(titles))
+        problem_dict = {}
+        for title in titles:
+            file_list = [item['@id'] for fileset in result.get('other_processed_files', [])
+                         for item in fileset['files'] if fileset['title'] == title]
+            file_list.extend([item['@id'] for exp in result.get('experiments_in_set', [])
+                              for fileset in exp['other_processed_files']
+                              for item in fileset['files'] if fileset['title'] == title])
+            statuses = set([status_dict[f] for f in file_list])
+            if len(statuses) > 1:
+                problem_dict[title] = {f: status_dict[f] for f in file_list}
+            elif 'release' not in result['status'] and (
```

Comment:
I think this is because 'released' and 'released to project' have the same status level, correct?  I have run into similar and I think we might generally simplify things if we set 'released to project' and 'pre-release' status levels as intermediate?  

Here is the status_level dictionary I added to the item release script that may do the trick but have not tested it in this case and other status audits as yet.  Might this be a useful approach? I guess it needs the 'archived' and 'archived to project' status levels tweaked for this purpose. 

STATUS_LEVEL = {
    # standard_status
    "released": 10, "current": 10,
    "released to project": 9,
    "pre-release": 8,
    "planned": 6, "submission in progress": 6,
    "in review by lab": 4,
    "revoked": 0, "archived": 0,"deleted": 0, "obsolete": 0, "replaced": 0, "archived to project": 0,
    # additional file statuses
    'to be uploaded by workflow': 4, 'uploading': 4, 'uploaded': 4, 'upload failed': 4, 'draft': 4, 'released to lab': 4}